daemon/daemon_main.go

```diff
@@ -322,6 +322,7 @@ func init() {
 
 	flags.Bool(option.DisableK8sServices, false, "Disable east-west K8s load balancing by cilium")
 	option.BindEnv(option.DisableK8sServices)
+	flags.MarkDeprecated(option.DisableK8sServices, "This option is no longer supported and will be removed in v1.9")
```

Comment:
Nit: we still support the option, so maybe `This option has been deprecated and will be removed in v1.9`?

Reply:
Sure will address your comments.